menpo/image/base.py

```diff
@@ -2525,17 +2530,38 @@ def pixels_range(self):
 
     def rolled_channels(self):
         r"""
+        Deprecated - please use the equivalent ``as_rolled_channels`` method.
+        """
+        warn('This method is no longer supported and will be removed in a '
+             'future version of Menpo. '
+             'Use .as_rolled_channels() instead.',
+             MenpoDeprecationWarning)
+        return self.as_rolled_channels()
+
+    def as_rolled_channels(self, out_dtype=None):
```

Comment:
Yes, to be fair, I've struggled with the naming convention of this the entire time. There is no standard term for where the channels lie and so everything might seem confusing to someone. *Rolling* is obviously what you actually do to move the axis. 'back' and 'front' are more obvious, but might be confusing to someone with regards to the 'back/front' of *what*? 'First' and 'last' axis are more explicit but lend themselves to longer names. It's tough, discoverability is important but super verbose method names for fairly common functions are also undesirable. 

`.pixels_array_with_channels_at_back()` is definitely too long. I don't think we need to specify `array` as we already have the convention that `pixels` gives access to the underlying numpy array. So I would vote for `.pixels_with_channels_at_back()`.

I can see why the `as_` syntax might suggest returning a Menpo object given the definition of similar methods. However, obviously this needs to be a method as it involves a copy. We also need to fix the `init_from_rolled_channels` to have consistent naming e.g. `init_from_channels_at_back()`.